src/base/static/js/utils.js

```diff
@@ -30,48 +30,16 @@ var self = (module.exports = {
     }
   },
 
-  // Determine whether or not the current logged-in user has admin rights
-  // for the passed datasetId. Returns true or false.
-  getAdminStatus: function(
-    datasetId,
-    adminGroups = [],
-    isSubmitterEditingEnabled = false,
-    submitter = null,
-  ) {
-    // First, check if the current user is the submitter of the place.
-    if (
-      Shareabouts.bootstrapped.currentUser &&
-      submitter &&
-      isSubmitterEditingEnabled &&
-      Shareabouts.bootstrapped.currentUser.username === submitter.username &&
-      Shareabouts.bootstrapped.currentUser.provider_id === submitter.provider_id
-    ) {
-      return true;
-    }
-
-    // Next, check if the current user is a member of an approved admin group.
-    if (
+  // Determine whether the current user is a member of the "administrators"
+  // group.
+  getAdminStatus: function(datasetId) {
```

Comment:
I think there is still a use case for a special group that confers abilities that other groups don't have (such as the ability to access the dashboard and download protected bulk data).

In this PR we assume that this group has the arbitrary name `administrators`.

Reply:
Or we can store the names of the groups that have access to the dashboard in our dashboard config. Same goes for our "download data" button. In this case, the group name would be `administrators`.

It would be great to kill this method and generalize permissions handling.

Reply:
I agree-- I think we can remove this method entirely in this PR.